scripts/entrypoint.sh

```diff
@@ -15,4 +16,10 @@ if [ -n "${LAUNCHPAD_KEYCLOAK_URL}" ]; then
     sed -i.bckp 's#realm:.*,clientId#realm:"'${LAUNCHPAD_KEYCLOAK_REALM}'",url:"'${LAUNCHPAD_KEYCLOAK_URL}'",clientId#' ${APP_JS}
 fi
 
+if [ -n "${LAUNCHPAD_TRACKER_SEGMENT_TOKEN}" ]; then
+    sed -i.bckp 's#SegmentTrackerToken#${LAUNCHPAD_TRACKER_SEGMENT_TOKEN}#' ${INDEX}
```

Comment:
I think you are missing some ' now it's replaced by `${LAUNCHPAD_TRACKER_SEGMENT_TOKEN}` instead of the value
you should change it to:
```
sed -i.bckp 's#SegmentTrackerToken#'${LAUNCHPAD_TRACKER_SEGMENT_TOKEN}'#' ${INDEX}
```